packages/houston-astro/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "houston-astro",
```

No reason this can't be be an official package from the `@astrojs` scope!

```suggestion
  "name": "@astrojs/houston-theme",
```